.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint",
```

Comment:
I would prefer Prettier than eslint to format code, and it wasn't part of ticket to setup formatters, next time ask me about such questions if ticket too short